lib/helpers/shorts/short_controller.dart

```diff
@@ -161,9 +151,7 @@ class ShortController {
     previousSeconds = progressSeconds;
 
     if (player.value.error != null && (currentSetupCompleter == null || currentSetupCompleter!.isCompleted)) {
-      debugPrint('SHRT: player error: ${player.value.error}, retries: $retries');
       if (retries > 3) {
-        debugPrint('SHRT: failed to play short ${s.id} after 3 retries. Player error: ${player.value.error}');
```

Comment:
Yea sorry about the logs :)
Some of these catch edge cases which can be quite useful for future debugging though, for example "retried 3 times, stopping", etc.
I would go through and consider if any of them would be worth having. From a quick skim looks like I'd only keep these two.
Can also be useful to create a custom logging function where you filter out based on a log-level.